### ISSUE TITLE
Include it in Kodi's repository of add-ons

### ISSUE THREAD

**Issue by PanderMusubi** [michaelrcarroll/script.kodi.hue.ambilight#24]
Please include it in Kodi's repository of add-ons:

http://kodi.wiki/view/Submitting_Add-ons

For at least version 15 and upwards. This would allow users to install this add-on directly from Kodi.

**Comment by michaelrcarroll** [comment]
Yep! This is on the list of things to do, but I want to go through a bit of testing of the 0.8 version first. Adding to the repo will *dramatically* increase the number of users, which if there are issues could mean an overwhelming amount of support. Be on the lookout for a 0.8 beta *very* soon to start testing over the holidays.

**Comment by PanderMusubi** [comment]
Super. By the way, you can ask for a code review on the Kodi forum. People there are very helpful and have a lot of experience with add-ons. Looking forward to seeing it included.

**Comment by PanderMusubi** [comment]
Is 0.8 available in https://github.com/mpolednik/script.kodi.hue.ambilight ?

If that is the leading repo, which issues and code needs to move over there?
